CHANGELOG.md

```diff
@@ -10,7 +10,9 @@ _None_
 
 ### New Features
 
-_None_
+* Added `xcfilelist` subcommand to generated input and output file lists to be used with Xcode's Run Script build phase.  
```

Comment:
This needs to be updated with the new command name, and I'd add the explanation from the description, that mentions it's meant for phases that execute `swiftgen config run`.

Something like:
```suggestion
* Added `config generate-xcfilelist` subcommand to generated input and output `xcfilelists` based on your configuration file. This should be used in an Xcode build step that executes `swiftgen config run`.  
```